docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -1222,6 +1225,48 @@ Set parameters for customizing the :term:`UPP`.
    
    Note that this variable is first changed to lower case before being used to construct the file names.
 
+RUN_PRDGEN Configuration Parameters
+=====================================
+
+Non-default parameters for the ``run_prdgen`` task are set in the ``task_run_prdgen:`` section of the ``config.yaml`` file.
+
+Basic Task Parameters
+---------------------------------
+For each workflow task, certain parameter values must be passed to the job scheduler (e.g., Slurm), which submits a job for the task.
+
+``TN_RUN_PRDGEN``: (Default: "run_prdgen")   Set the name of this Rocoto workflow task. Users typically do not need to change this value.
+
+``NNODES_RUN_PRDGEN``: (Default: 1) 
+   Number of nodes to use for the job.
+
+``PPN_RUN_PRDGEN``: (Default: 1)
+   Number of :term:`MPI` processes per node.
+
+``WTIME_RUN_PRDGEN``: (Default: 00:30:00)
+   Maximum time for the task to complete.
+
+``MAXTRIES_RUN_PRDGEN``: (Default: 2)
+   Maximum number of times to attempt the task.
+
+``KMP_AFFINITY_RUN_PRDGEN``: (Default: "scatter")
+   Intel Thread Affinity Interface for the ``run_prdgen`` task. See :ref:`this note <thread-affinity>` for more information on thread affinity.
+
+``OMP_NUM_THREADS_RUN_PRDGEN``: (Default: 1) 
+   The number of OpenMP threads to use for parallel regions.
+
+``OMP_STACKSIZE_RUN_PRDGEN``: (Default: "1024m")
+   Controls the size of the stack for threads created by the OpenMP implementation.
+
+``DO_PARALLEL_PRDGEN``: (Default: false)
+   Flag that determines whether to use CFP to run the product generation job in parallel.  This should be used with the RRFS_NA_3km grid and PPN_RUN_PRDGEN should be set to 22.
```

Comment:
@BenjaminBlake-NOAA What is CFP? Better to define for newbies even if seasoned forecasters would know. 
Also, could you add backticks around RRFS_NA_3km grid and PPN_RUN_PRDGEN?
Thanks! :)

Reply:
Sure I can add backticks to those two variables.  CFP allows you to run multiple executions of code at the same time, but I can't seem to find what it actually stands for!  Will attempt to track that down...